crates/figma_import/Cargo.toml

```diff
@@ -19,6 +19,7 @@ fetch = ["clap"]
 "serde_json" = "1.0"
 "serde_bytes" = "0.11"
 "bincode" = "1.3"
+"thiserror" = "1.0"
```

Comment:
This is good -- `thiserror` and `anyhow` are the crates for syntax sugaring error handling in Rust that are recommended by Core Android.